integration_tests/test_math.py

```diff
@@ -149,6 +150,14 @@ def test_trunc():
     i = trunc(-4.5)
     assert i == -4
 
+def test_modf():
+    i: f64
+    i = 3.14
+
+    res: tuple[f64, i32]
+    res = modf(i)
+    assert res[0] == 0.14000000000000012
```

Comment:
I think we should tests like, `assert res[0] - 0.14 <= 1e-6`

Reply:
Seems like we don't have ability to index tuples. :-(

```bash
semantic error: Type mismatch in comparison operator, the types must be compatible
```

Reply:
EDIT: `assert abs(res[0] - 0.14) <= 1e-6`